packages/storybook-vue/stories/3_components/radio-button-group/RadioButtonGroup.stories.mdx

```diff
@@ -0,0 +1,268 @@
+        <scale-radio-button
+          value="1"
+          label="Radio Button"
+          name="radioStandardOne"
+          transparent="true"
+          input-id="choiceOne"
+        ></scale-radio-button>
+        <div style="margin-top: 0.5rem"></div>
+        <scale-radio-button
+          value="2"
+          label="Radio Button"
+          name="radioStandardTwo"
+          input-id="choiceTwo"
+        ></scale-radio-button>
+      </div>
+    </scale-radio-button-group>
+    </div>
+  `,
+});
+
+export const ErrorTemplate = (args, { argTypes }) => ({
+  props: {
+    width: { type: String },
+    ...ScaleRadioButtonGroup.props,
+  },
+  computed: {
+    wrapperStyles() {
+      return {
+        maxWidth: this.width,
+      };
+    },
+  },
+  template: `
+    <div :style="wrapperStyles">
+   <scale-radio-button-group
+      label="Group Label"
+      helper-text="Hilfe es gab einen Fehler"
+    >
+      <div slot="radio-button-list">
+        <scale-radio-button
+          value="1"
+          status="error"
+          label="Radio Button"
+          name="radioError"
+          transparent="true"
+          input-id="choiceOne"
+        ></scale-radio-button>
+        <div style="margin-top: 0.5rem"></div>
+        <scale-radio-button
+          value="2"
+          status="error"
+          label="Radio Button"
+          name="radioError"
+          input-id="choiceTwo"
+        ></scale-radio-button>
+      </div>
+    </scale-radio-button-group>
+    </div>
+  `,
+});
+
+## Standard
+
+<Canvas withSource="none">
+  <Story
+    name="Standard"
+    args={{
+      label: 'RadioButton Group',
+    }}
+  >
+    {StandardTemplate.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Standard" />
+
+```html
+<scale-radio-button-group label="Group Label">
+  <div slot="radio-button-list">
+    <scale-radio-button
+      value="1"
+      label="Radio Button"
+      name="radioStandard"
+      transparent="true"
+      input-id="choiceOne"
+    ></scale-radio-button>
+    <div style="margin-top: 0.5rem"></div>
+    <scale-radio-button
+      value="2"
+      label="Radio Button"
+      name="radioStandard"
+      input-id="choiceTwo"
+    ></scale-radio-button>
+  </div>
+</scale-radio-button-group>
+```
+
+## One Selectable
+
+<Canvas withSource="none">
+  <Story
+    name="OneSelect"
+    args={{
+      label: 'RadioButton Group',
+    }}
+  >
+    {StandardTemplate.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-radio-button-group label="Group Label">
+  <div slot="radio-button-list">
+    <scale-radio-button
+      value="1"
+      label="Radio Button"
+      name="radioStandard"
+      transparent="true"
+      input-id="choiceOne"
+    ></scale-radio-button>
+    <div style="margin-top: 0.5rem"></div>
+    <scale-radio-button
+      value="2"
+      label="Radio Button"
+      name="radioStandard"
+      input-id="choiceTwo"
+    ></scale-radio-button>
+  </div>
+</scale-radio-button-group>
+```
+
+## All Selectable
```

Comment:
Is this part of the requirements? If so, we might need some sort of state to handle this (, and an Eventemitter to emit this state).